implementations/micrometer-registry-stackdriver/build.gradle

```diff
@@ -3,7 +3,7 @@ dependencies {
 
     api 'com.google.cloud:google-cloud-monitoring'
     implementation 'org.slf4j:slf4j-api'
-    implementation 'ch.qos.logback:logback-classic'
+    compileOnly 'ch.qos.logback:logback-classic'
```

Comment:
Should it perhaps be a test dependency?

Reply:
IMHO, It should be another module and compile dependency. If it's intended to be shipped to end users. If it's just for internal use - yes, test module also works.

Reply:
Or just delete manual level change, idk.

Reply:
It's used in the little utility class `ClearCustomMetricDescriptors`, but OK to move to `compileOnly` as you suggest.